src/providers/rekey-search/rekey-search.ts

```diff
@@ -13,7 +13,8 @@ export class RekeySearchProvider {
     private urlProvider: UrlProvider,
   ) { }
 
-  getTest(rekeySearchParams: RekeySearchParams): Observable<TestSlot> {
+  getTest(rekeySearchParams: RekeySearchParams): Observable<Object> {
+    // return of(this.zippedMockTestData);
```

Comment:
commented code

Reply:
Yeah, there are 2 feature which needs to be developed, view CandidateDetails and start the Rekey process. I would like to keep this here so that it's easier to test locally.
Will make sure to remove it once the features are there